tls/s2n_connection.h

```diff
@@ -275,6 +275,10 @@ struct s2n_connection {
     s2n_ct_support_level ct_level_requested;
     struct s2n_blob ct_response;
 
+    /* QUIC transport parameters data: https://tools.ietf.org/html/draft-ietf-quic-tls-29#section-8.2 */
+    struct s2n_blob our_quic_transport_parameters;
+    struct s2n_blob peer_quic_transport_parameters;
```

Comment:
It's kind of a bummer that all non-quic connections have to pay the cost of these fields. But I get that we didn't want to complicate the build with a bunch of flags .. trade-offs.

Reply:
We also keep blobs around on the connection for other optional extensions that aren't commonly used, so this isn't particularly unusual.